Gemfile.lock

```diff
@@ -499,38 +503,38 @@ GEM
     omniauth-rails_csrf_protection (1.0.1)
       actionpack (>= 4.2)
       omniauth (~> 2.0)
-    pagy (3.6.0)
-    parallel (1.19.2)
-    parallel_tests (2.21.1)
+    pagy (3.14.0)
+    parallel (1.24.0)
+    parallel_tests (2.32.0)
       parallel
     parser (2.7.2.0)
       ast (~> 2.4.1)
-    path_expander (1.0.4)
-    pg (1.1.3)
-    pry (0.10.3)
+    path_expander (1.1.1)
+    pg (1.5.4)
+    pry (0.12.2)
       coderay (~> 1.1.0)
-      method_source (~> 0.8.1)
-      slop (~> 3.4)
+      method_source (~> 0.9.0)
     pry-byebug (3.3.0)
       byebug (~> 8.0)
       pry (~> 0.10)
-    pry-rails (0.3.5)
-      pry (>= 0.9.10)
-    pry-rescue (1.4.5)
+    pry-rails (0.3.9)
+      pry (>= 0.10.4)
+    pry-rescue (1.6.0)
       interception (>= 0.5)
-      pry
+      pry (>= 0.12.0)
     pry-stack_explorer (0.4.9.3)
       binding_of_caller (>= 0.7)
       pry (>= 0.9.11)
-    public_suffix (5.0.1)
+    public_suffix (5.0.4)
     puma (5.6.8)
       nio4r (~> 2.0)
     pyu-ruby-sasl (0.0.3.3)
     racc (1.7.3)
     rack (2.2.8)
-    rack-mini-profiler (1.1.4)
+    rack-mini-profiler (1.1.6)
```

Comment:
### [rack-mini-profiler](https://miniprofiler.com)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.1.4 → 1.1.6

[_[change-log](https://github.com/MiniProfiler/rack-mini-profiler/blob/master/CHANGELOG.md), [source-code](https://github.com/MiniProfiler/rack-mini-profiler), [gem-diff](https://my.diffend.io/gems/rack-mini-profiler/1.1.4/1.1.6)_]





<details>
<summary>Commits</summary>

A change of **5** commits. See the full changes on [the compare page](https://github.com/MiniProfiler/rack-mini-profiler/compare/v1.1.4...v1.1.6).

These are the individual commits:
- (930eccd) [FIX: handle storage engine failures in whitelist mode](https://github.com/MiniProfiler/rack-mini-profiler/commit/930eccd99d5e4acf8ff43558b06a9a4ed134d5f8)
- (d62736e) [FIX: Custom counters regression, and stop console errors whe…](https://github.com/MiniProfiler/rack-mini-profiler/commit/d62736ecb4f519b5048bf4a3435ba1f1228a3d01)
- (90e1a04) [bump version 1.1.5](https://github.com/MiniProfiler/rack-mini-profiler/commit/90e1a040b5502b9872c8f3c733d72287e3fd655a)
- (10f3ac6) [FIX: if totalsControl has no parent do not explode](https://github.com/MiniProfiler/rack-mini-profiler/commit/10f3ac61ce5457dbf08cd715e478def471e693e9)
- (2a8211b) [bump version](https://github.com/MiniProfiler/rack-mini-profiler/commit/2a8211bf2efd7e3e692d1fffa969b7f495570bd3)


</details>